miaodx;changes to make things a little better and some instructions to use on windows(10_x64).

MiaoDX opened the pull request:
Hope you accept pull request,in fact,this is my first try to raise a PR.^_^

1.some changes to the `pom.xml`
2.more specific instructions to use in windows(10)
3.add a picture to attract attention
4.update the readme file
5.upload some `.gml` files

MiaoDX committed:
miaodx;change to make things a little better.^_^
1.some changes to the `pom.xml`
2.more specific instructions to use in windows(10)
3.add a picture to attract attention
4.update the readme file
5.upload some `.gml` files

Commit:
miaodx;update readme when I try again in my laptop.

fcarsten reviewed (changes requested):
Thanks for your contribution. Generally happy to include them with the exception of some of the changes to the README file. I would like this file to give the reader a short introduction into what the software is and maybe how to install and use it. It should not contain comments, suggestions, etc. These should be captured elsewhere. If you could tidy up the README file a bit I'd be happy to merge your changes.

Commit:
make README.md more clean.

MiaoDX commented:
Hei, @fcarsten 
It does looks a little districtive with a long README. Changed a little and moved notes about run in windows into `windows_helper.md`, hope it's ok now.

MiaoDX commented:
It has been a long time, and I have lost the context of this PR, so I will close it now, good luck:) .
